tests/__init__.py

```diff
@@ -34,13 +34,13 @@
 
 if not os.environ.get('GOOGLE_APPLICATION_CREDENTIALS'):
     os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = get_gcp_credentials_file().name
-with open(os.environ['GOOGLE_APPLICATION_CREDENTIALS'], "r") as fh:
-    os.environ['ADMIN_USER_EMAILS'] = json.loads(fh.read())['client_email']
-bundles.ADMIN_USER_EMAILS = set(os.environ['ADMIN_USER_EMAILS'].split(","))
+bundles.ADMIN_USER_EMAILS = get_admin_user_emails()
```

Comment:
We should be able to pull `ADMIN_USER_EMAILS` into the test suite like this:
`os.environ['ADMIN_USER_EMAILS'] = ""`

Reply:
changed to recommended method